### PR TITLE
Tammy review

### PR DESCRIPTION
**Pull request by tammytamayao**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    ],
+  },
+  resolve: {
+    extensions: ['.tsx', '.ts', '.js'],
+  },
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+};
```
Comment (rafael-parayno) [inline webpack.config.js:20]: add new line

File: src/sessions/SignUp.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            <div>
+            <button type="submit">Register</button><br/>
+            Already a user? <Link to="/login"> Sign in here. </Link>
+            </div>
+        </form>
+    </>
+  )
+}
+
+export default SignUp
```
Comment (rafael-parayno) [inline src/sessions/SignUp.tsx:58]: add new line

File: src/sessions/LogIn.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                <div>
+                <button type="submit">Log In</button><br/>
+                Not yet a user? <Link to="/"> Sign up here. </Link>
+                </div>
+            </form>
+        </>
+      )
+}
+
+export default LogIn
```
Comment (rafael-parayno) [inline src/sessions/LogIn.tsx:59]: add new line

File: src/components/Post.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      <h3>
+        <Link to="/posts"> Back </Link> |
+        <Link to={`/posts/edit/${params.id}`}> Edit </Link> |
+        <button type="button" className="btn btn-danger" onClick={deletePost}>Delete Post</button>
+      </h3>
+    </div>
+    </>
+  );
+}
+export default Post;
```
Comment (rafael-parayno) [inline src/components/Post.tsx:56]: add new line

File: src/components/NewPost.tsx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+            <button type="submit">Create Post</button> <br/>
+            <Link to="/posts">Back to Main</Link>
+
+          </form>
+    </div>
+  );
+};
+
+export default NewPost;
```
Comment (rafael-parayno) [inline src/components/NewPost.tsx:89]: add new line

File: src/components/EditPost.tsx [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+    
+                <button type="submit">Update Post</button> <br/>
+                <Link to="/posts">Back to Main</Link>
+    
+              </form>
+        </div>
+    )
+};
+
+export default EditPost;
```
Comment (rafael-parayno) [inline src/components/EditPost.tsx:109]: add new line

File: src/components/EditPost.tsx [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+        setFunction(post.target.value);
+    };
+
+    var titleDef = post.title;
+    var bodyDef = post.body;
+
+    const onSubmit = (post: React.FormEvent<HTMLFormElement>) => {
+      
+        post.preventDefault();
+        console.log(post)
```
Comment (rafael-parayno) [inline src/components/EditPost.tsx:41]: do we need this console.log. remove this

File: src/components/NewPost.tsx [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+
+  const onChange = (post: React.ChangeEvent<HTMLInputElement>, setFunction: { (value: React.SetStateAction<string>): void; (value: React.SetStateAction<string>): void;}) => {
+    setFunction(post.target.value);
+  };
+
+  const onSubmit = (post: React.FormEvent<HTMLFormElement>) => {
+    post.preventDefault();
+    const url = process.env.REACT_APP_API_ACTIVE+`api/v1/posts/create`;
+
+    fetch(url, {
```
Comment (rafael-parayno) [inline src/components/NewPost.tsx:31]: can you use axios for this. so then add axios interceptor. so we dont need to always add the base url.

**Review by rafael-parayno — CHANGES_REQUESTED** [review]

File: src/components/EditPost.tsx (hunk 7 above)
Comment (tammytamayao) [reply src/components/EditPost.tsx]: done

File: src/sessions/LogIn.tsx (hunk 3 above)
Comment (tammytamayao) [reply src/sessions/LogIn.tsx]: done

File: src/sessions/SignUp.tsx (hunk 2 above)
Comment (tammytamayao) [reply src/sessions/SignUp.tsx]: done

File: src/components/Post.tsx (hunk 4 above)
Comment (tammytamayao) [reply src/components/Post.tsx]: done

File: src/components/EditPost.tsx (hunk 6 above)
Comment (tammytamayao) [reply src/components/EditPost.tsx]: done

File: webpack.config.js (hunk 1 above)
Comment (tammytamayao) [reply webpack.config.js]: done

File: src/components/NewPost.tsx (hunk 5 above)
Comment (tammytamayao) [reply src/components/NewPost.tsx]: done

File: src/components/NewPost.tsx (hunk 8 above)
Comment (tammytamayao) [reply src/components/NewPost.tsx:30]: done. Changes pushed to new branch named AxiosConfig

**Review by rafael-parayno — APPROVED** [review]

**Pull request merged**